code/game/gamemodes/blob/overmind.dm

```diff
@@ -87,13 +87,15 @@
 	if (!message)
 		return
 
-	var/message_a = say_quote("\"[html_encode(message)]\"")
+	var/message_a = say_quote("\"[message]\"")
```

Comment:
No, it's calling decode.
`#define log_blobspeak(text) diary << html_decode("\[[time_stamp()]]BLOB: [text]")`